libgadget/cooling.h

```diff
@@ -30,6 +30,7 @@ struct cooling_units
 
 /*Initialise the cooling module.*/
 void init_cooling(char * TreeCoolFile, char * MetalCoolFile, char * reion_hist_file, struct cooling_units cu, Cosmology * CP);
+double rho_crit_baryon;
```

Comment:
Could you please make this a member of cooling_units?

Reply:
Done, is this what you had in mind?